github_review_helper_suite_test.go

```diff
@@ -187,3 +200,34 @@ var PullRequestEvent = func(action, headSHA string, headRepository grh.Repositor
   }
 }`
 }
+
+var createStatusEvent = func(sha, state string, branches []grh.Branch) string {
+	branchSHAs := make([]string, len(branches))
+	for i, branch := range branches {
+		branchSHAs[i] = branch.SHA
+	}
+	return `{
+  "id": 214015194,
```

Comment:
Copied from example payload from GitHub docs. Not used in our code, so could be removed, but also doesn't do harm.